CODE_OF_CONDUCT.md

```diff
@@ -211,56 +235,93 @@ Potential consequences for violating the conda Organization code of conduct incl
+| ---- | ---- | ---- |
+| Within 24 Hours | Acknowledge | The CoC Committee will make every effort to **acknowledge receipt of a report within 24 hours** (and we'll aim for much more quickly than that). |
+| Within 3 Days | Review | The CoC Committee will make all efforts to **review the incident within three days**. |
+| Within 1 Week | Resolve | We aim to **respond within one week** to the original reporter with either a resolution or an explanation of why the situation is not yet resolved. |
+
+# Voting and Decision Making
+
+Committee votes and decisions require both a minimum quorum size for the vote to be counted, and then a minimum percentage of cast affirmative votes to pass.
+
+Except where otherwise noted, votes require a quorum and a simple majority to pass:
+
+* Minimum Quorum:
+  * More than 50% of eligible committee members must vote.
+    * Eligibile members do not include those excluded because of [conflicts of interest](#conflicts-of-interest).
+* Affirmative vote threshold:
+  * More than 50% of the votes cast need to be affirmative to take action.
+
+# Committee Membership
+
+You can reach the entire CoC Committee by emailing [condaconduct@gmail.com](mailto:condaconduct@gmail.com).
+
+| Name | Contact | Employer / Funding | Steering Council Member | Current Term Ends |
+| ---- | ---- | ---- | ---- | --- |
+| [Dave Clements](https://github.com/tnabtaf) | dcl&#101;ments&#64;anaconda&#46;com, +1 541 914 6324 | [Anaconda](https://anaconda.com/) | | 2023-07-01 |
+| [Mahe Iram Khan](https://github.com/ForgottenProgramme) | | [Anaconda](https://anadonda.com/) | | 2023-07-01 | 
+| [Ken Odegard](https://github.com/kenodegard) | | [Anaconda](https://anaconda.com/) | | 2023-07-01 |
+| [Crystal Soja](https://github.com/csoja), Co-Chair | | [Anaconda](https://anaconda.com/) | Yes | 2023-07-01 |
+| [Eric Dill](https://github.com/ericdill), Co-Chair | | [Voltron Data](https://voltrondata.com/) | Yes | 2024-07-01 |
+| [Filipe Pires Alvarenga Fernandes](https://github.com/ocefpaf) | ocefpaf@gmail.com | [IOOS/NOAA](https://ioos.noaa.gov/) | Yes | 2024-07-01 |
+| [Bianca Henderson](https://github.com/beeankha) | | [Anaconda](https://anaconda.com/) | | 2024-07-01 |
+| [Katherine Kinnaman](https://github.com/kathatherine) | kkinnaman.condacoc@gmail.com | [Anaconda](https://anadonda.com/) | | 2024-07-01 | 
+| [](), [External Ombudsperson](#ombudsperson) | | [External Org]() | | 2024-07-01 |
+
+# Terms and New Members
+
+* Committee members are appointed for two year terms. Committee members can choose to renew their memberships.
+* Committee members can resign before their term ends.
+* Committee members can also be removed by a [simple majority vote](#voting-and-decision-making) from their fellow committee members.
+* New committee members are added by a simple majority vote as well.
+
+# Eligibility
+
+Anyone from the community who is interested and able to do CoC Committee work is eligible to be nominated for the committee.  
```

Comment:
Is self-nomination allowed?

Reply:
It should be!  I will make that clearer.

Reply:
Thank you!